Completely disable external libedit support. When libedit is installed, the
check will automatically add -ledit to LIBS and the result will be wrong
strlcpy check results and linking failures.

https://bugs.gentoo.org/198906

--- tnftp-20070806/configure.ac
+++ tnftp-20070806/configure.ac
@@ -84,10 +84,7 @@
                       [termcap termlib curses ncurses tinfo],
                       [],
                       [AC_MSG_ERROR(
-                           [no relevant library found containing tgetent])])
-       AC_SEARCH_LIBS([el_init],
-                      [edit],
-                      [have_libedit=yes], [have_libedit=no])])
+                           [no relevant library found containing tgetent])])])
 
 AC_SEARCH_LIBS([gethostbyname], [nsl])
 AC_SEARCH_LIBS([socket],
